tests/test_misc.py

```diff
@@ -0,0 +1,12 @@
+import os
+from pathlib import Path
+
+
+def test_create_leftover():
+    """Create a leftover dataset to simulate a failed test, to verify test isolation."""
+    dataset_dir = os.environ.get(
+        "MINARI_DATASETS_PATH",
+        os.path.join(os.path.expanduser("~"), ".minari/datasets/"),
+    )
+    new_dataset_path = Path(dataset_dir) / "leftover_dataset/data"
+    new_dataset_path.mkdir(parents=True)
```

Comment:
I would remove this test before merging as we don't really need to test for it after